qa-ibd.planx-pla.net/portal/gitops.json

```diff
@@ -94,18 +94,18 @@
           "color": "#a2a2a2",
           "name": "Files"
         },
-        {
-          "icon": "query",
-          "link": "/query",
-          "color": "#a2a2a2",
-          "name": "Query"
-        },
         {
           "icon": "workspace",
           "link": "#hostname#workspace/",
           "color": "#a2a2a2",
           "name": "Workspace"
         },
+        {
+          "name": "Notebook Browser",
```

Comment:
maybe place it right after "exploration", instead of "files", which we truly don't need?